dev/src/platform.cpp

```diff
@@ -568,32 +568,36 @@ void SetConsole(bool on) {
     #endif
 }
 
-iint LaunchSubProcess(const char **cmdl, const char *stdins, string &out) {
+iint LaunchSubProcess(const char** cmdl, const char* stdins, string& out) {
```

Comment:
please undo, here and elsewhere

Reply:
Do I need to change (char* const) to (char *const) as well?

Reply:
yes